### PR TITLE
added the hovering effect to the buttons which were not under the hov…

### LINKED ISSUE #389: Buttons on talawa-admin show different behaviour on hovering over them(few buttons are responsive to hovering while few of tht buttons aren't)
**Describe the bug**
Few buttons on the portal have hover properties while few dont respond to hovering over them

**To Reproduce**

1.To the talwa-admin portal and try to hover on the login or register button,You will see no hover effect
2.Whereas when u reach to the dashboard after logging in, there is this another button named as "create organizations" which responses to hovering

**Expected behavior**
All the buttons should have hovering properties to make make user experince better.

**Actual behavior**
few buttons are responsive to hoverness while few aren't.

**Screenshots**
[Screencast from 20-01-23 01:09:36 AM IST.webm](https://user-images.githubusercontent.com/107215525/213545034-794b72ae-224c-452d-8139-a7ec3f4cd899.webm)
the video shows how hovering on the lower button shows responsiveness while upper button doesn't show any response.

### PR DESCRIPTION
**Pull request by anurag0006**
…ering effect + made the coloring scheme consistant for all the buttons which had different colors

**What kind of change does this PR introduce?**

**Issue Number:**
#389 
Fixes #

**Did you add tests for your changes?**
No(not needed)

**Snapshots/Videos:**
image
Earlier "createOrgButton" had different color from the given color scheme.

**If relevant, did you update the documentation?**
Not needed

**Summary**
The Pr bring all the buttons under the hovering effect and also fixing the inconsistancies in the color scheme of some buttons/components and hence Closes #389 .

**Does this PR introduce a breaking change?**
No

**Other information**
Not needed

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please read CONTRIBUTING.md on how to auto-close issues when PRs are approved and merged. You will need to edit the first comment of the PR.

**Comment by anurag0006** [comment]
@palisadoes , i have Added the required keyword for closing the related issue in the summary section of the PR...
image

Should i still edit the first comment of the PR to added the keyword again for auto-closing the issue?

**Comment by palisadoes** [comment]
That should be OK. I didn't see it before.

**Comment by Kevoniat** [comment]
hi @anurag0006   the code looks good to me

**Review by palisadoes — APPROVED** [review]

File: src/screens/LoginPage/LoginPage.module.css [hunk 1]
```diff
@@ -111,6 +111,11 @@ form > input {
   cursor: pointer;
   transition: transform 0.2s, box-shadow 0.2s;
   width: 100%;
+}
+.whiteloginbtn:hover{
+background-color:#737373;
+color:white;
```
Comment (Kevoniat) [inline src/screens/LoginPage/LoginPage.module.css:117]: some consistency would be nice so either use the color name or the hex code ( i prefer the hex code )

**Review by Kevoniat — APPROVED** [review]

**Pull request merged**